Doc/library/math.rst

```diff
@@ -224,11 +224,11 @@ Number-theoretic and representation functions
    of *x* and are floats.
 
 
-.. function:: nextafter(x, y)
+.. function:: nextafter(x, y, steps=1)
```

Comment:
`steps=1` is a bit of a lie: the real default is `None`, but the code treats `None` the same as 1.

Reply:
Yep, that seems fine to me.